lesson3/lesson3.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include <math.h>
+
+int main()
+{
+    //First task
+    int Time = 0;
+    int Hours = 0;
+    int Minutes = 0;
+    int Seconds = 0;
+    std::cout << "Enter how many time you spent in game in seconds: ";
+    std::cin >> Time;
+
+    Hours = Time / 3600;
```

Comment:
Consider using const variables for const values:
const unsigned SecondsPerMinute = 60;
const unsigned MinutesPerHour = 60;
const unsigned SecondsPerHour = SecondsPerMinute * MinutesPerHour;

Reply:
Added const values